Buildings/HeatTransfer/Conduction/MultiLayer.mo

```diff
@@ -10,9 +10,22 @@ model MultiLayer
     "Heat flow rate from state i to i+1";
   extends Buildings.HeatTransfer.Conduction.BaseClasses.PartialConstruction;
 
+  parameter Boolean placeCapacityAtSurf_a=false
+    "Set to true to place the capacity at the surface a of the layer"
+    annotation (Dialog(tab="Dynamics"),
+                Evaluate=true);
+  parameter Boolean placeCapacityAtSurf_b=false
+    "Set to true to place the capacity at the surface b of the layer"
+    annotation (Dialog(tab="Dynamics"),
+                Evaluate=true);
+  parameter Integer nSta2[nLay]={layers.material[i].nSta for i in 1:nLay}
+    "Vector of number of states per material layer" annotation(Evaluate=true);
 protected
   Buildings.HeatTransfer.Conduction.SingleLayer[nLay] lay(
+   nSta2={nSta2[i] for i in 1:nLay},
    each final A=A,
+   final placeCapacityAtSurf_a = {if i == 1 then placeCapacityAtSurf_a else false for i in 1:nLay},
```

Comment:
@tsnouidui : It is best if you use `sed` and check the diff afterwards. There are also some comments in the `info` section that would need to be updated so the documentation writes about states rather than capacity.